otlp/go.mod

```diff
@@ -1,10 +1,10 @@
 module go.opentelemetry.io/proto/otlp
 
-go 1.15
+go 1.14
```

Comment:
Is this a needed change? I would like to maintain compatibility with the same minimum version supported by opentelemetry-go if possible.

Reply:
I believe opentelemetry-go targets [1.14](https://github.com/open-telemetry/opentelemetry-go/blob/main/go.mod#L3). This change brings it in line with the SDK.

Reply:
Oh shoot, you're right. I misread the diff 👍